ci/d2-convert-tag.sh

```diff
@@ -29,5 +24,5 @@ then
 	set +x
 	git -c "credential.https://github.com.username=${OCEAN_D2_USER}" \
 		-c "core.askPass=./ci/askpass.sh" \
-		push "https://github.com/sociomantic-tsunami/ocean-d2.git" "$d2tag"
+		push "https://github.com/sociomantic-tsunami/ocean.git" "$d2tag"
```

Comment:
I think it is nice to keep as it immediately tells that content was auto-generated. Is it documented anywhere?